packages/smarthr-ui/src/components/Textarea/Textarea.tsx

```diff
@@ -217,13 +266,16 @@ export const Textarea = forwardRef<HTMLTextAreaElement, Props & ElementProps>(
     return maxLetters ? (
       <span>
         {body}
-        <span className={counterStyle} id={actualMaxLettersId}>
-          {beforeMaxLettersCount}
-          <span className={counterTextStyle}>
-            {maxLetters - count}/{maxLetters}
+        <span className={counterStyle}>
+          <span className={counterTextStyle} id={actualMaxLettersId} aria-hidden={true}>
+            {counterVisualMessage}
           </span>
-          {afterMaxLettersCount}
         </span>
+
+        <VisuallyHiddenText aria-live="polite">{srCounterMessage}</VisuallyHiddenText>
+        <VisuallyHiddenText id={maxLettersNoticeId}>
+          こちらは文字カウンター付きの入力欄です。最大{maxLetters}文字まで入力できます。
```

Comment:
```suggestion
          最大{maxLetters}文字入力できます。
```

「最大◯文字入力できます」だけ言えば通じると思うので、最小限にするとよさそう。